src/pages/AttendanceRecordPage.tsx

```diff
@@ -35,10 +35,12 @@ const AttendanceRecordPage = () => {
   const fetchMemberRecord = async (): Promise<Event[]> => {
     // Update the member from the API
     const member = await fetchMember(userID!);
```

Comment:
I don't think we need these fetchMember calls anymore, since now we have the authContext and can access the logged in members data there. I think we can remove all instances of the old context (LoginContext in app.tsx) because all it did was store the uuid of the logged in member so we made another call, but the member data is already stored. 

Side note, not sure how the stored jwt data would behave with a function that updates the logged in Member data (specifically on the userPreference page to update the notPresesntEmail boolean), the data could become a little stale but its not a big issue, this could something to backlog.

Reply:
hm good point, the data will get old. maybe it does make sense to still fetch the user instead of store it in jwt